### PR TITLE
Apply updates from latest CitA spreadsheet

### PR DESCRIPTION
**Pull request by stevenwilkin**
We will need to go through the proposed changes to the titles and address unfortunately :unamused:

### CODE REVIEW

**Comment by stevenwilkin** [comment]
Rebased

File: public/js/cita.json [hunk 1]
```diff
@@ -4680,7 +4680,7 @@
     "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
   },
   {
-    "title": "Chester-le-Street",
+    "title": "Alzheimers Society (Durham) ",
```
Comment (andrewgarner) [inline public/js/cita.json]: These appear to be duplicates?

Comment (stevenwilkin) [inline public/js/cita.json]: Different addresses though?

Comment (andrewgarner) [inline public/js/cita.json]: Addressed with 2ca5dd61185998b02a2ce79c9edecaa1ad4c6bec

Comment (stevenwilkin) [inline public/js/cita.json]: :heart_eyes_cat:

File: public/js/cita.json [hunk 2]
```diff
@@ -3544,7 +3544,7 @@
     "title": "Brent",
     "address": "270-272 High Road\nLONDON\nNW10 2EY",
     "phone": "020 84381240",
-    "hours": "Monday to Friday, 10am to 4pm",
+    "hours": "Monday to Friday, 9.30am to 4.30pm",
```
Comment (stevenwilkin) [inline public/js/cita.json]: _9.30am_ or _9:30am_ ?

Comment (andrewgarner) [inline public/js/cita.json]: 9:30am - I'll fix and rebase

**Comment by andrewgarner** [comment]
@stevenwilkin how does this look now?

**Comment by stevenwilkin** [comment]
Magic! Interesting to note that the vast majority of title updates we've had to discard. I'll get this out the door now and inform CitA.

**Pull request merged**